secrets-provider-test-chart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: secrets-provider-test-chart
+version: 1.0.0
```

Comment:
We might want to add `kubeVersion` also: https://helm.sh/docs/topics/charts/#the-kubeversion-field

Reply:
We can also add `home` with `secrets provider` github repo url.

Reply:
@eladkug which version of k8s are we supporting? (not oc, k8s)